data/_models/shared/Materials.csv

```diff
@@ -5,9 +5,29 @@ Materials,,produced by process,ref_array,,,,Processes,,,,http://purl.obolibrary.
 Materials,,individuals,ref_array,,,,Individuals,,,,http://purl.obolibrary.org/obo/NCIT_C164337,A reference to one or more individuals associated to the material,"Patient registry, FAIR Genomes",
 Materials,,alternate ids,string_array,,,,,,,,http://purl.obolibrary.org/obo/NCIT_C90353,Additional identifiers that identify a material,"Patient registry, FAIR Genomes",
 Materials,,included in resources,ref_array,,,,Resources,,,,http://purl.obolibrary.org/obo/HSO_0000370,"A reference to one or more resources that organises materials for in a meaningful way (e.g., data releases, subpopulations, cohorts, etc.)","Patient registry, FAIR Genomes",
+Files,Materials,,,,,,,,,,"http://purl.obolibrary.org/obo/STATO_0000002,http://purl.obolibrary.org/obo/NCIT_C42883","An electronic file is an information content entity which conforms to a specification or format and which is meant to hold data and information in digital form, accessible to software agents.",Patient registry,
+Files,,file locations,refback,,,,File locations,,file,,,,Patient registry,
+Files,,md5 checksum,string,,,,,,,,http://purl.obolibrary.org/obo/NCIT_C171276,A 32-character hexadecimal number that is computed on a file.,Patient registry,
+Files,,format,ontology,,TRUE,CatalogueOntologies,File formats,,,,"http://purl.org/dc/terms/FileFormat,http://purl.obolibrary.org/obo/NCIT_C171252","A digital resource format (e.g. BAM, FastQ, gVCF, phenopacket, BED)",Patient registry,
+Files,,biosamples,ref_array,,,,Biosamples,,,,http://purl.obolibrary.org/obo/NCIT_C43376,A reference to one or more biosamples that are associated with this file,Patient registry,
+Files,,generated by protocol,ref_array,,,,Protocol activity,,,,,,Patient registry,
```

Comment:
For consistency, it might be better to define everything in materials first followed by files.

Reply:
Or does this address the issue where the columns of the inherited columns appeared first?